week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,6 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+	return firstWord + secondWord + thirdWord; // Write the body of this function to concatenate three words together
```

Comment:
Yep. This will work. Everyone else used this function for their solution:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat
You might like to look at their pull requests to see what they did. Actually I like yours because it is easy to understand.